tests/playwright/src/model/pages/volumes-page.ts

```diff
@@ -36,6 +36,12 @@ export class VolumesPage extends MainPage {
     this.collectUsageDataButton = this.additionalActions.getByRole('button', { name: 'Gather volume sizes' });
   }
 
+  async kindClusterVolumeExists(): Promise<boolean> {
+    const volumesTable = await this.getTable();
+    const kindClusterVolume = volumesTable.getByRole('row').first();
+    return (await kindClusterVolume.count()) > 0;
+  }
+
```

Comment:
Can you clarify on what this does? How is it different from just getting the first row in the volumes page?

Reply:
All available methods require at least one parameter, which is the volume name. Since a volume created with the cluster has a different name each time, I need a method that can check for the existence of any volume. I also attempted to extract volume names from the container's inspect data but I didn't get it.

Reply:
ok, but how do you know the first volume is the volume for the kind cluster? What if the kind cluster volume is 2nd or 5th in the list?